grgit-gradle/src/main/groovy/org/ajoberstar/grgit/gradle/GrgitPlugin.groovy

```diff
@@ -22,15 +22,14 @@ class GrgitPlugin implements Plugin<Project> {
         grgit.close()
       }
 
-      project.allprojects { prj ->
-        if (prj.ext.has('grgit')) {
+      project.allprojects { Project prj ->
+        if (prj.extensions.hasProperty('grgit')) {
           prj.logger.warn("Project ${prj.path} already has a grgit property. Remove org.ajoberstar.grgit from either ${prj.path} or ${project.path}.")
         }
-        prj.ext.grgit = grgit
+        prj.extensions.add(Grgit,"grgit", grgit)
```

Comment:
There might be some behavior change between `ext` vs `extensions`.

We had some places where we added `grgit` plugin multiple times. It was working fine before. With this change, `grgit` within subprojects becomes `null`.

Applying only at the root level solves it. 

The issue is that it is really hard to debug. Here is the error we get:

```
> Cannot invoke method fetch() on null object
```